tmux/record/record_ros_bag.yml

```diff
@@ -19,7 +19,7 @@ root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
 # on_project_start: command
 on_project_start: |
   export ROS_NAMESPACE=${ROS_NAMESPACE:-/kitware}
-  export HL2_IP=${HL2_IP:-192.168.1.101}
+  export HL2_IP=${HL2_IP:-192.168.1.4}
```

Comment:
Why is the IP of the Hololens not '.101'?

Reply:
Here at KRS it is .4 on the wifi.... I can keep the default .101 but maybe it's better for the default to be what we use on Remy?

Reply:
sure, leave it at 4 then.